ambry-quota/src/main/java/com/github/ambry/quota/capacityunit/AmbryCUQuotaEnforcer.java

```diff
@@ -113,9 +116,9 @@ public void shutdown() {
    * @return QuotaRecommendation object.
    */
   private QuotaRecommendation buildQuotaRecommendation(float usage, QuotaName quotaName) {
-    boolean shouldThrottle = (usage >= 100);
-    return new QuotaRecommendation(shouldThrottle, usage, quotaName,
-        shouldThrottle ? throttleRetryAfterMs : NO_THROTTLE_RETRY_AFTER_MS);
+    QuotaAction quotaAction = (usage >= 100) ? QuotaAction.DELAY : QuotaAction.ALLOW;
```

Comment:
Consider replacing the 100 with a config param. In the future, we may let users to exceed their quota up to a certain point

Reply:
Added it as a final variable for now, instead of hard coding.
The main reason why I am hesitant to create a config is that it adds one more config, that wouldn't be used now, which will make it confusing.